Source/ZXing.Net.Mobile.Forms.WindowsUniversal/ZXing.Net.Mobile.Forms.WindowsUniversal.csproj

```diff
@@ -131,7 +131,9 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
-    <Reference Include="netstandard"/>
+    <Reference Include="netstandard">
+      <HintPath>Libs\netstandard.dll</HintPath>
```

Comment:
adding this binary lib is a bit ugly (and doesn't seem to work anyway), how about you revert this commit and try to add the nuget package called "NETStandard.Library"

Reply:
Unfortunately, it doesn't work with that nuget package 😢. It gives the same error as before. That's a nasty workaround but it works. The best solution would be to update target project to [newer ones](https://stackoverflow.com/questions/51211144/the-type-attribute-is-defined-in-an-assembly-that-is-not-referenced-xamarin) but I think that's a task for another PR.